llvm/utils/TableGen/CallingConvEmitter.cpp

```diff
@@ -34,12 +34,12 @@ class CallingConvEmitter {
 public:
   explicit CallingConvEmitter(const RecordKeeper &R) : Records(R) {}
 
-  void run(raw_ostream &o);
+  void run(raw_ostream &O);
```

Comment:
nit: can this be `OS`. I've seen that as commonly used variable name for raw_ostream?

Reply:
Do you want me to change all `&O` to `&OS` in the file? There are quite a few methods in this file that have raw_ostream as parameter and they are all defined to be `&O`

Reply:
Yes, since you are changing the, you might as well use the preferred name

Reply:
@jurahul should I create a new PR for this change? It looks like there are few other files that also needs this modification. I think it might introduce too many changed files in this PR.